week-4/day-1/03supermarketQueue.rb

```diff
@@ -0,0 +1,20 @@
+def queue_time(customers, n)
+  checkout = Array.new(n, 0)
+
+  if customers.empty?
+    return 0
+  else
+    customers.each do |x|
+      checkout[0] = checkout[0] + x
```

Comment:
Can you explain to me what this line of code does?

Reply:
Almost forgot about this. =S
It's an accumulator which is fed by the numbers in the customers array. The checkout is another array which is ordered in ascending order.
The idea here is to get the minimum time (values in customers array) given a number of checkout boxes (size of the checkout array). The minimum time is the biggest number in the checkout array.